src/components/Avatar/Avatar.js

```diff
@@ -0,0 +1,15 @@
+import styled from "styled-components";
+import { prop } from "styled-tools";
+import as from "../../enhancers/as";
+import Base from "../Base";
+
+const Avatar = styled(Base)`
+  border-radius: 50%;
+  width: 1.5em;
```

Comment:
imho would be cool if it accepted a special prop of `size` that would set automatically width/height

Reply:
For now, I think `fontSize` is enough. We can add that in the future when people come up with use cases.